src/ontology/uberon-edit.obo

```diff
@@ -223716,30 +223708,27 @@ xref: UBPROP:0000002
 is_metadata_tag: true
 
 [Typedef]
-id: bearer_of
-name: bearer of
+id: has_characteristic
```

Comment:
Is this relationship used in Uberon? If not, remove it better.

Reply:
Yes, it's being used in many terms.

Reply:
but if you change the id, dont you have to change 1000 other things in the uberon-edit.obo file? Best not do all these changes in this PR: I would recommend leaving the `id` as is and only changing the `name:`? Or what do you think?

Reply:
Ah no, sorry, I was confused by another property that I only changed the name.